features/table/utils/filter-converter.ts

```
@@ -0,0 +1,281 @@
+        rangeConditions.push({
+          data: {
+            path: [id],
+            gte: value1,
+          },
+        } as Prisma.RecordWhereInput);
+      }
+      if (value2 !== null) {
+        rangeConditions.push({
+          data: {
+            path: [id],
+            lte: value2,
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)